docs/root/operations/admin.rst

```diff
@@ -309,7 +309,7 @@ modify different aspects of the server:
 
   - To change the logging level across all loggers, set the query parameter as level=<desired_level>.
   - To change a particular logger's level, set the query parameter like so, <logger_name>=<desired_level>.
-  - To change multiple logging levels at once, set the query parameter as paths=<logger_name1>=<desired_level1>,<logger_name2>=<desired_level2>.
+  - To change multiple logging levels at once, set the query parameter as paths=<logger_name1>:<desired_level1>,<logger_name2>:<desired_level2>.
```

Comment:
```suggestion
  - To change multiple logging levels at once, set the query parameter as ``paths=<logger_name1>:<desired_level1>,<logger_name2>:<desired_level2>``.
```
i think these probably should be literals - i think it might make the text a bit less confusing, but i can see the file has others similar - so up to you on updating those